bai2/helpers.py

```diff
@@ -17,6 +43,22 @@ def _build_record(rows):
     return Record(code=rows[0][0], fields=fields, rows=rows)
 
 
+RecordBuilderFactory = {
+    RecordCode.file_header: _build_generic_record,
+    RecordCode.group_header: _build_generic_record,
+    RecordCode.account_identifier: _build_account_identifier_record,
+    RecordCode.transaction_detail: _build_generic_record,
+    RecordCode.account_trailer: _build_generic_record,
+    RecordCode.group_trailer: _build_generic_record,
+    RecordCode.file_trailer: _build_generic_record,
+}
```

Comment:
Does `RecordCode.continuation` need to be listed here too??

Reply:
I think that `RecordCode.continuation` isn't parsed by itself, but only together with one of the other types. That is the reason we have several rows when parsing (there is one the record codes and its continuation rows)